<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>UIMarginsPropertyGridWidget</class>
 <widget class="QWidget" name="UIMarginsPropertyGridWidget">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>300</width>
    <height>110</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>300</width>
    <height>110</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <widget class="QLabel" name="label_14">
   <property name="geometry">
    <rect>
     <x>152</x>
     <y>53</y>
     <width>51</width>
     <height>20</height>
    </rect>
   </property>
   <property name="minimumSize">
    <size>
     <width>0</width>
     <height>0</height>
    </size>
   </property>
   <property name="text">
    <string>Bottom:</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignVCenter</set>
   </property>
  </widget>
  <widget class="QLabel" name="label_7">
   <property name="geometry">
    <rect>
     <x>152</x>
     <y>23</y>
     <width>31</width>
     <height>20</height>
    </rect>
   </property>
   <property name="minimumSize">
    <size>
     <width>0</width>
     <height>0</height>
    </size>
   </property>
   <property name="text">
    <string>Top:</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignVCenter</set>
   </property>
  </widget>
  <widget class="QLabel" name="label_6">
   <property name="geometry">
    <rect>
     <x>5</x>
     <y>23</y>
     <width>31</width>
     <height>20</height>
    </rect>
   </property>
   <property name="minimumSize">
    <size>
     <width>0</width>
     <height>0</height>
    </size>
   </property>
   <property name="text">
    <string>Left:</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignVCenter</set>
   </property>
  </widget>
  <widget class="QDoubleSpinBox" name="rightDoubleSpinBox">
   <property name="geometry">
    <rect>
     <x>45</x>
     <y>50</y>
     <width>80</width>
     <height>25</height>
    </rect>
   </property>
   <property name="decimals">
    <number>1</number>
   </property>
   <property name="minimum">
    <double>-9999.000000000000000</double>
   </property>
   <property name="maximum">
    <double>9999.000000000000000</double>
   </property>
   <property name="singleStep">
    <double>1.000000000000000</double>
   </property>
  </widget>
  <widget class="QDoubleSpinBox" name="leftDoubleSpinBox">
   <property name="geometry">
    <rect>
     <x>45</x>
     <y>20</y>
     <width>80</width>
     <height>25</height>
    </rect>
   </property>
   <property name="decimals">
    <number>1</number>
   </property>
   <property name="minimum">
    <double>-9999.000000000000000</double>
   </property>
   <property name="maximum">
    <double>9999.000000000000000</double>
   </property>
   <property name="singleStep">
    <double>1.000000000000000</double>
   </property>
  </widget>
  <widget class="QLabel" name="label_8">
   <property name="geometry">
    <rect>
     <x>5</x>
     <y>53</y>
     <width>40</width>
     <height>20</height>
    </rect>
   </property>
   <property name="minimumSize">
    <size>
     <width>0</width>
     <height>0</height>
    </size>
   </property>
   <property name="text">
    <string>Right:</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignVCenter</set>
   </property>
  </widget>
  <widget class="QDoubleSpinBox" name="topDoubleSpinBox">
   <property name="geometry">
    <rect>
     <x>203</x>
     <y>20</y>
     <width>80</width>
     <height>25</height>
    </rect>
   </property>
   <property name="decimals">
    <number>1</number>
   </property>
   <property name="minimum">
    <double>-9999.000000000000000</double>
   </property>
   <property name="maximum">
    <double>9999.000000000000000</double>
   </property>
   <property name="singleStep">
    <double>1.000000000000000</double>
   </property>
  </widget>
  <widget class="QDoubleSpinBox" name="bottomDoubleSpinBox">
   <property name="geometry">
    <rect>
     <x>203</x>
     <y>50</y>
     <width>80</width>
     <height>25</height>
    </rect>
   </property>
   <property name="decimals">
    <number>1</number>
   </property>
   <property name="minimum">
    <double>-9999.000000000000000</double>
   </property>
   <property name="maximum">
    <double>9999.000000000000000</double>
   </property>
   <property name="singleStep">
    <double>1.000000000000000</double>
   </property>
  </widget>
  <widget class="QPushButton" name="resetMarginsButton">
   <property name="geometry">
    <rect>
     <x>44</x>
     <y>80</y>
     <width>225</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text">
    <string>Reset Margins</string>
   </property>
  </widget>
  <widget class="QLabel" name="MarginsLabel">
   <property name="geometry">
    <rect>
     <x>1</x>
     <y>0</y>
     <width>62</width>
     <height>16</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <weight>50</weight>
     <italic>false</italic>
     <bold>false</bold>
     <kerning>true</kerning>
    </font>
   </property>
   <property name="text">
    <string>Margins:</string>
   </property>
  </widget>
 </widget>
 <tabstops>
  <tabstop>leftDoubleSpinBox</tabstop>
  <tabstop>topDoubleSpinBox</tabstop>
  <tabstop>rightDoubleSpinBox</tabstop>
  <tabstop>bottomDoubleSpinBox</tabstop>
  <tabstop>resetMarginsButton</tabstop>
 </tabstops>
 <resources/>
 <connections/>
</ui>
